lib/statsd/instrument/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module StatsD
   module Instrument
-    VERSION = "2.4.0"
+    VERSION = "2.3.9"
```

Comment:
You shouldn't have to lie anymore in Shopify core now that I've resolve the version conflict.